packages/CalendarCarousel/README.md

```diff
@@ -1,9 +1,72 @@
-# calendar
+# Calendar Carousel
+
+## Description
+
+This is a Calendar Carousel that displays each month. It automatically moves on to the next year so you can swipe infinitely through the dates. 
+
+#### Features of this Calendar Include
+Display Current Date with a blue circle
+Display Selected Date with a light blue circle
+Scroll Through each of the months (will automatically load to the next year on December)
+
+
+#### Language
+- React-Native : FlatList, StyleSheet, SafeAreaView, Text, Texstyle...
+- typescript 
+
+#### Libraries
+Date-fns : https://date-fns.org/
```

Comment:
I don't see you are using it! Also, it is best not to have it inside.

Reply:
okey doke!